packages/app-desktop/services/spellChecker/SpellCheckerServiceDriverNative.ts

```diff
@@ -17,43 +16,25 @@ export default class SpellCheckerServiceDriverNative extends SpellCheckerService
 		return this.session().availableSpellCheckerLanguages;
 	}
 
-	// Language can be set to '' to disable spell-checking
-	public setLanguage(v: string) {
+	// Language can be set to [] to disable spell-checking
+	public setLanguage(v: string[]) {
```

Comment:
setLanguages

Reply:
This will cause different methods names in `SpellCheckerServiceDriverBase` and `SpellCheckerServiceDriverNative`, and they are related. It will cause linting errors. Do you really want this change?

Reply:
Consider changing it to `setLanguages` in `SpellCheckerServiceDriverBase`.

Reply:
Yes please let's change to setLanguages. Sorry about that extra trouble @antontkv, I'd rather we get this right in this PR if that's ok with you.